packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -1372,6 +1370,55 @@ class SemanticsNode extends AbstractNode with DiagnosticableTreeMixin {
     _dirty = false;
   }
 
+  /// Builds a new list made of [_children] sorted in semantic traversal order.
+  List<SemanticsNode> _childrenInTraversalOrder() {
+    final List<SemanticsNode> childrenInDefaultOrder = _childrenInDefaultOrder(_children);
```

Comment:
do we have to do all this work to bring nodes, that specify a sort key, into default sort order only to re-sort them then according to their sort key?